Plugin/globals.cpp

```diff
@@ -761,7 +760,7 @@ static void DoReadProjectTemplatesFromFolder(const wxString& folder, std::list<P
         }
     }
 
-    if (loadDefaults && files.IsEmpty()) {
+    if (loadDefaults && list.empty()) {
```

Comment:
Previously, files was always empty.
I suspect you wanted to check `list` from below comment.
For previous behavior, remove ` && files.IsEmpty()` instead.